itou/templates/api/index.html

```diff
@@ -0,0 +1,53 @@
+{% extends "layout/content.html" %}
+
+{% block title %}API Les emplois de l’inclusion{% endblock %}
+
+{% block extra_head %}
+    {{ block.super }}
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet">
+    <link rel="shortcut icon" href="/static/favicon.ico" type="image/ico">
+    <link crossorigin="anonymous" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" rel="stylesheet">
+    <script crossorigin="anonymous" integrity="sha384-ZvpUoO/+PpLXR1lu4jmpXWu80pZlYUAfxl5NsBMWOEPSjUn/6Z/hRTt8+pR6L4N2" src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+<script crossorigin="anonymous" integrity="sha384-Piv4xVNRyMGpqkS2by6br4gNJ7DXjqk09RmUpJ8jgGtD7zP9yug3goQfGII0yAns" src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js"></script>
+{% endblock %}
+
+{% block content %}   
+    <div class="container">
+       <div class="row no-gutters pt-3">
+           <div class="col col-lg-10 offset-lg-1 mt-4">
+            <div class="jumbotron">
+                <h1>API Les emplois de l’inclusion</h1>
+                <p class="lead">L'API des <a href="{% url 'home:hp' %}">emplois de l'inclusion</a> met à disposition des données de la plateforme.</p>
+                <p>Cet API est encore en construction, et est appellée à évoluer.</p>
+                <p>
+                <p class="text-muted">Version: 1</p>
+            </div>
+            <div class="row mt-5">
+                <div class="col-12 bg-info text-white p-3">
+                    L’accès à certaines ressources peut nécessiter une authentification avec des droits particuliers.<br />
+                    Si vous désirez obtenir ces droits sur votre compte, merci de nous contacter.
```

Comment:
On donne le lien vers le support ? Ou un mail direct ?

```suggestion
                    Si vous désirez obtenir ces droits sur votre compte, merci de <a href="{{ ITOU_ASSISTANCE_URL }}" target="_blank" rel="noopener">nous contacter</a>{% include "includes/icon.html" with icon="external-link" %}.
```

Reply:
Bonne idée. J’ai finalement privilégié:
 - l’adresse mail `contact` (la page assistance ne permet pas d’entrer en contact directement)
 - une classe un peu différente, le lien en bleu foncé sur bleu pale est difficilement lisible

![Sélection_105](https://user-images.githubusercontent.com/1223316/134008691-0647134e-228b-438e-b9bd-909574c63d35.png)